packages/ses/scripts/bundle.js

```diff
@@ -2,20 +2,21 @@ import '../index.js';
 import fs from 'fs';
 import { makeBundle } from '@endo/compartment-mapper/bundle.js';
 import terser from 'terser';
+import { fileURLToPath, pathToFileURL } from 'url';
 
-const resolve = (rel, abs) => new URL(rel, abs).toString();
-const root = resolve('..', import.meta.url);
+const resolve = (rel, abs) => fileURLToPath(new URL(rel, abs));
```

Comment:
It feels like `resolve` is not really longer necessary and can be inlined where used?

Reply:
I could go either way on this. It is a mouthful, but I’ve inlined it elsewhere habitually.